docs/desktop-wpf/migration/convert-project-from-net-framework.md

```diff
@@ -120,107 +120,107 @@ Once the NuGet packages are updated to recent versions, the `<PackageReference>`
+The third step of the porting process is getting the project to build. For some apps, they will already build successfully once the project file is converted to an SDK-style project. If that's the case for your app, congratulations! You can go on to step 4. Other apps will need some updates to get them building for .NET Core. If you try to run `dotnet build` on the Bean Trader sample project now, for example, (or build it in VS), there will be about 100 errors, but you'll get them fixed up quickly.
 
 ### System.ServiceModel references and Microsoft.Windows.Compatibility
 
-A common source of errors is missing references for APIs that are available for .NET Core but not automatically included in the .NET Core App metapackage. To address this, you should reference the [Microsoft.Windows.Compatibility] package. The compatibility package includes a broad set of APIs that are common in Windows desktop apps such as WCF client, directory services, registry, configuration, ACLs APIs, and more.
+A common source of errors is missing references for APIs that are available for .NET Core but not automatically included in the .NET Core App metapackage. To address this, you should reference the `Microsoft.Windows.Compatibility` package. The compatibility package includes a broad set of APIs that are common in Windows desktop apps such as WCF client, directory services, registry, configuration, ACLs APIs, and more.
 
-With the Bean Trader sample, the majority of the build errors are due to missing System.ServiceModel types. These could be addressed by referencing the necessary WCF NuGet packages. WCF client APIs are among those present in the [Microsoft.Windows.Compatibility] package, though, so referencing the compatibility package is an even better solution (since it also addresses any issues related to APIs besides the WCF ones that the compatibility package makes available). The [Microsoft.Windows.Compatibility] package helps in most .NET Core 3 WPF and WinForms porting scenarios. After adding the NuGet reference to Microsoft.Windows.Compatibility, only one build error remains!
+With the Bean Trader sample, the majority of the build errors are due to missing System.ServiceModel types. These could be addressed by referencing the necessary WCF NuGet packages. WCF client APIs are among those present in the `Microsoft.Windows.Compatibility` package, though, so referencing the compatibility package is an even better solution (since it also addresses any issues related to APIs besides the WCF ones that the compatibility package makes available). The `Microsoft.Windows.Compatibility` package helps in most .NET Core 3 WPF and WinForms porting scenarios. After adding the NuGet reference to Microsoft.Windows.Compatibility, only one build error remains!
```

Comment:
```suggestion
With the Bean Trader sample, the majority of the build errors are due to missing <xref:System.ServiceModel> types. These could be addressed by referencing the necessary WCF NuGet packages. WCF client APIs are among those present in the `Microsoft.Windows.Compatibility` package, though, so referencing the compatibility package is an even better solution (since it also addresses any issues related to APIs besides the WCF ones that the compatibility package makes available). The `Microsoft.Windows.Compatibility` package helps in most .NET Core 3 WPF and WinForms porting scenarios. After adding the NuGet reference to Microsoft.Windows.Compatibility, only one build error remains!
```